introduction/java_loops_i/Solution.java

```diff
@@ -0,0 +1,14 @@
+import java.util.Scanner;
+
+public class Solution {
+  private static final int NUMBER_OF_ITERATIONS = 10;
+  public static void main(String[] args) {
+    Scanner scan = new Scanner(System.in);
+    int inputNumber = scan.nextInt();
+    for (int iterationNumber = 1; iterationNumber <= NUMBER_OF_ITERATIONS; iterationNumber++) {
+      int resultOfMultiplication;
```

Comment:
Generally `int a = b;` is preferred compared to 

    int a;
    a = b;

That is declare and assign value in the same expression if possible.